tests/std/tests/P0226R1_math_special_functions/test.cpp

```diff
@@ -0,0 +1,499 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <cassert>
+#include <cmath>
+#include <limits>
+
+using namespace std;
+
+template <typename Float>
+void assert_close(const Float f, const Float g) {
+    auto eps = 0.0001 * abs(f);

```

Comment:
This is still an insanely large epsilon; can you use `std::numeric_limits<Float>::epsilon()`?

Reply:
Where can I get expected values?
I took examples from https://en.cppreference.com/w/cpp/numeric/special_functions/assoc_legendre and other cppreference pages.
But the precision of the examples is small, so tests fail with `16 * std::numeric_limits<Float>::epsilon()`